### DISCUSSION TITLE
Refresh not working

### DISCUSSION THREAD

**Discussion by Jertheguy992** [LittleBigRefresh/Refresh#551] [Q&A/Support]
So i recently restarted my pc before I restarted it the refresh for windows would work fine, but now it is saying its done but it does not open. I would like some help with this issue.

**Comment by Beyley** [comment]
Can you please elaborate what you mean by "does not open"? what exactly are you opening and what are you trying to do?

**Reply by Jertheguy992** [reply]
K So it opened up the terminal loaded and after saying "Ready to go! took ###ms" it said "Blocking in multithreaded mode with 4 tasks." and stayed on that the entire time. I think ive stayed on that for like 10 minutes. Edit: I did open up the lbp refresh app to add refresh to a game.
